atlascope/core/models/tour.py

```diff
@@ -0,0 +1,16 @@
+# from typing import List
+from uuid import uuid4
+
+from django.db import models
+from rest_framework import serializers
+
+
+class Tour(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
+    way_points = models.ManyToManyField('Waypoint', through='Investigation')
```

Comment:
I don't think this is right--we don't want to use `Investigation` as the through-table. I'm thinking we want to invent a new model to handle the relationship between several `Waypoint`s and a `Tour`. Check out the [membership example](https://docs.djangoproject.com/en/4.0/ref/models/fields/#django.db.models.ManyToManyField.through) in the Django docs. Let's discuss in person if you want to dig into this.

Reply:
resolved with 56d7e61af852626496a99eb9a5c4ec4b7b136f08